src/index.ts

```diff
@@ -127,10 +127,9 @@ async function reloadConfig(noteId: string) {
   const valid = ymlConfig !== null && (await board.loadConfig(ymlConfig));
   if (valid) {
     openBoard = board;
-  } else {
-    openBoard = undefined;
```

Comment:
wouldn't it be good to set openBoard as undefined in this case?

Reply:
Remove the code to prevent the board from closing automatically and keep the current board working.

Reply:
If it set to undefined, I am afraid the current board will not be working any more.

Reply:
Ok, I'll need more time to understand the code on my side then

Reply:
Ok, after additional test, this part is ok